prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -17,7 +17,7 @@ if [[ -z ${CCDB_POPULATOR_UPLOAD_PATH+x} ]]; then
   if [[ $RUNTYPE == "SYNTHETIC" ]]; then
     CCDB_POPULATOR_UPLOAD_PATH="http://ccdb-test.cern.ch:8080"
   elif [[ $RUNTYPE == "PHYSICS" ]]; then
-    CCDB_POPULATOR_UPLOAD_PATH="http://ccdb-test.cern.ch:8080"
+    CCDB_POPULATOR_UPLOAD_PATH="http://alice-ccdb.cern.ch"
```

Comment:
Should not it be `http://o2-ccdb.internal` and only in `EPNSYNCMODE==1` ?

Reply:
Or even `http://localhost:8084`, right @costing ?
And yes, only in `EPNSYNCMODE==1`

Reply:
If this runs in the Online environment, then yes, `http://localhost:8084` should be the target. Or `http://o2-ccdb.internal` (where the redirects will go to anyway). Clearly **not** `alice-ccdb.cern.ch` because that endpoint will require a valid X.509 certificate to upload objects to.

Reply:
Changed, please x-check.